packages/cadl-ranch-specs/http/azure/client-generator-core/client-format/duration/main.tsp

```diff
@@ -0,0 +1,90 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+import "@azure-tools/typespec-client-generator-core";
+
+using TypeSpec.Http;
+using Azure.ClientGenerator.Core;
+
+@doc("Test for clientFormat decorator on duration.")
+@supportedBy("dpg")
+@scenarioService("/azure/client-generator-core/client-format/duration")
+namespace _Specs_.Azure.ClientGenerator.Core.ClientFormat.Duration;
+
+@global.Azure.ClientGenerator.Core.operationGroup
+@route("/query")
+namespace Query {
+  @route("/iso8601")
+  @scenario
+  @scenarioDoc("""
+  Test that default client format for a duration parameter is iso8601.
+  Expected query parameter `input=P40D`
+  """)
+  op iso8601(
+    @query
+    input: duration
+  ): NoContentResponse;
+
+  @route("/seconds")
+  @scenario
+  @scenarioDoc("""
+  Test seconds client format for a duration parameter.
+  Expected query parameter `input=35.6`
+  """)
+  op seconds(
```

Comment:
We'll need multiple routes for different formats - int and double.

Reply:
Will add the test after tcgc update.